### ISSUE TITLE
ENH: Develop energy-normalization so spectrogram robustly detects rhythmicity

### ISSUE THREAD

**Issue by asoplata** [jonescompneurolab/hnn-core#942]

**Comment by asoplata** [comment]
Question @ntolley @dylansdaniels : will this allow spectrogram-plotting of our shorter, default ERP simulation in the GUI? I ask because currently, y'all agreed that the spectrogram NOT plotting for the default GUI simulation is expected behavior (see https://github.com/jonescompneurolab/hnn-core/issues/937#issuecomment-2479703548 ). However, in our ERP tutorial (here https://jonescompneurolab.github.io/hnn-tutorials/coregui_erp/coregui_erp.html ), which uses the default GUI simulation settings, there is a specific section *"4.3 Viewing ERP Spectrograms"* about these spectrograms. Currently, these plots will not be generated for reasons in the first link. Whatever changes Nick derives to our spectral analysis, I think we should probably ensure it works in the shorter ERP case as well.

**Comment by dylansdaniels** [comment]
Good catch! I think we can safely remove that section from the tutorial. It's really just there to show off the GUI functionality, but it's not important or relevant for that tutorial.
